tests/integration/src/test/scala/mesosphere/marathon/integration/MesosCheckIntegrationTest.scala

```diff
@@ -0,0 +1,69 @@
+package mesosphere.marathon.integration
+
+import java.util.UUID
+
+import mesosphere.AkkaIntegrationTest
+import mesosphere.marathon.integration.facades.MarathonFacade.extractDeploymentIds
+import mesosphere.marathon.integration.setup.EmbeddedMarathonTest
+import mesosphere.marathon.raml.{App, AppCheck, CommandCheck, ShellCommand}
+import mesosphere.marathon.state.PathId
+
+class MesosCheckIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonTest {
```

Comment:
Maybe we can combine this with `HealthCheckIntegrationTest` ? Rename to "CheckIntegrationTest" ? There's a startup / teardown cost for each suite... seems like maybe we should optimize for this if we can?